<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeCondition</class>
 <widget class="QWidget" name="AttributeCondition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>36</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnstretch="0,0,0,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="1">
      <widget class="QComboBox" name="leftValueCombo"/>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="operatorCombo"/>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="rightValueLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>2000</width>
         <height>27</height>
        </size>
       </property>
       <property name="inputMask">
        <string notr="true"/>
       </property>
       <property name="placeholderText">
        <string>enter value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="deleteConditionButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/Resources.qrc">
         <normaloff>:/icon/DeleteCondition</normaloff>:/icon/DeleteCondition</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
